src/pages/distroSettings/NewDistroButton.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
```

Comment:
idk how you feel about this but I ended up putting `DistroSelect` in its own folder — does it make sense to put these components in a folder too? or you can un-folder-ify the `DistroSelect` if that's better